python/unblob/sandbox.py

```diff
@@ -8,7 +8,9 @@
 from structlog import get_logger
 
 from unblob._rust.sandbox import (
-    AccessFS,
+    AccessFS as AccessFS,
+)
+from unblob._rust.sandbox import (
```

Comment:
I don't understand why you can do `from ._rust` in `__init__.py` and you need to do `from unblob._rust` in this one.

Reply:
True, relative import would work in both cases.